RecoEgamma/EgammaElectronProducers/python/gsfElectronCores_cfi.py

```diff
@@ -18,4 +18,11 @@
     pfSuperClusterTrackMap = cms.InputTag("pfElectronTranslator:pf")
 )
 
+ecalDrivenGsfElectronCoresFromMultiCl = ecalDrivenGsfElectronCores.clone()
```

Comment:
same comment about cfi files
gsfElectronModules_cff seems more appropriate.
Also, phase2_hgcal.toModify for it is not really necessary and parameters can be passed in the clone method arguments